test/basic.html

```diff
@@ -350,6 +350,16 @@
           expect(customField.inputs[1]).to.equal(timePicker);
         });
 
+        it('custom field should not have has-value by default', () => {
+          expect(customField.hasAttribute('has-value')).to.be.false;
+        });
+
+        it('custom field should have has-value when value is provided', () => {
+          timePicker.value = '15:00';
+          dateTimePicker.__triggerCustomFieldValueUpdate();
+          expect(customField.hasAttribute('has-value')).to.be.true;
+        });
+
```

Comment:
Maybe also add a test for the improvement I suggested above related to has-value.

Something like `it('should not have has-value on custom field if only date or time is selected',` (or separate test for date and time). This would mean that also the existing latter test above would need to set both date and time value.

nit: would change wording on the other two also to `it('should (not) have has-value on custom field ...',` so it's more logical to read.

Reply:
Updated.